### DISCUSSION TITLE
How to add flag to get warning during coding?

### DISCUSSION THREAD

**Discussion by ghibli613** [microsoft/vscode-cpptools#9646] [Q&A]
When I was typing and miss `;` or `{`, red linter show up with some error messages in problem panel immediately. I just want to know how to get warning like that, in **Sublime Text** they have sublime-linter and I can add flag to get both warning and error linter in real-time coding. I only got waring linter and message after I had compiled my file with some flag. For example I got non-warning with `int n = 3; int a[n];` until I compiled  my file with `-Wall -O2` flags defined in _task.json_.

**Comment by sean-mcmanus** [comment]
Do you have C_Cpp.intelliSenseMode set to "Default"?
